installer/README.md

```diff
@@ -0,0 +1,102 @@
+## Volcano
+
+Volcano is a batch system built on Kubernetes. It provides a suite of mechanisms currently missing from
+Kubernetes that are commonly required by many classes of batch & elastic workload including:
+
+1. machine learning/deep learning,
+2. bioinformatics/genomics, and 
+3. other "big data" applications.
+
+## Prerequisites
+
+- Kubernetes 1.12+ with CRD support
+
+## Installing volcano via yaml file
+
+All-in-one yaml has been generated for quick deployment. Try command:
+```$xslt
+kubectl apply -f volcano-v0.0.x.yaml
```

Comment:
Add a guide on how to generate this yaml

Reply:
It should not be here, since it's only used for the releases and focus on how to use not how to develop.

Reply:
a step : make xxxx

Reply:
Since the source code is not downloaded in this scenario, it's better to add this steps into developing process.